app/services/staff_action_logger.rb

```diff
@@ -744,6 +744,33 @@ def log_unpublished_page(topic_id, slug)
     ))
   end
 
+  def log_add_email(user, opts = {})
```

Comment:
This is minor but I think it is worth specifying the keys that needs to be passed in so that we always log with the right information.

Reply:
Yes, I agree, You can use named args here.

Reply:
`opts` is no longer used after I removed `old_value` and `new_value` as they may contain sensitive user information (email addresses).